src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,651 @@
+"""migrate translation data - notices and experiences
+
+Revision ID: 14acee6f5459
+Revises: 0c65325843bd
+Create Date: 2024-01-09 21:17:13.115020
+
+"""
+
+import csv
+import uuid
+from datetime import datetime
+from enum import Enum
+
+import yaml
+from alembic import op
+from loguru import logger
+from sqlalchemy import text
+from sqlalchemy.engine import ResultProxy
+
+# revision identifiers, used by Alembic.
+
+revision = "14acee6f5459"
+down_revision = "0c65325843bd"
+branch_labels = None
+depends_on = None
+
+
+def generate_record_id(prefix):
+    return prefix + "_" + str(uuid.uuid4())
+
+
+# this ties a DB record ID to our logical identifier of the type of OOB experience.
+# the DB record ID is based on what we've defined in the associated config yml.
+class DefaultExperienceConfigTypes(Enum):
+    EEA_PRIVACY_CENTER = 1
+    EEA_MODAL_AND_BANNER = "pri-c8ff-78d6-4a02-850f-2c09dda-over-config"
+    EEA_TCF_OVERLAY = "d7c3ce0a-a3b3-43ff-bf6f-3d8-tcf-over-config"
+    US_MODAL = "pri-76b8-cc52-11ee-9eaa-8ef97a0-over-config"
+    US_PRIVACY_CENTER = "pri-27d4-cc53-11ee-9eaa-8ef97a04-pri-config"
+
+
+class ComponentType(Enum):
+    PrivacyCenter = "privacy_center"
+    Overlay = "overlay"
+    TCFOverlay = "tcf_overlay"
+
+
+# our new OOB EEA (EU) experience configs, keyed by their component type
+EEA_MAPPING = {
+    ComponentType.PrivacyCenter: DefaultExperienceConfigTypes.EEA_PRIVACY_CENTER,
+    ComponentType.Overlay: DefaultExperienceConfigTypes.EEA_MODAL_AND_BANNER,
+    ComponentType.TCFOverlay: DefaultExperienceConfigTypes.EEA_TCF_OVERLAY,
+}
+
+# our new OOB US experience configs, keyed by their component type
+US_MAPPING = {
+    ComponentType.PrivacyCenter: DefaultExperienceConfigTypes.US_PRIVACY_CENTER,
+    ComponentType.Overlay: DefaultExperienceConfigTypes.US_MODAL,
+}
+
+# this map contains the biggest _assumption_ about the migration -
+# these are the notices that will be associated with our new OOB experience configs.
+# any existing notices besides these will remain "orphaned" post-migration.
+# additionally, the regions that (pre-migration) are associated with these notices
+# will be used to infer the regions associated with the post-migration experience
+# configs to which they map.
+NOTICES_TO_EXPERIENCE_CONFIG = {
+    "marketing": EEA_MAPPING,
+    "functional": EEA_MAPPING,
+    "essential": EEA_MAPPING,
+    "analytics": EEA_MAPPING,
+    "data_sales_and_sharing": US_MAPPING,
+}
+
+
+# as part of this upgrade, we're removing the following as region options across the app
+REGIONS_TO_REMOVE = {
+    "gb_eng",
+    "gb_sct",
+    "gb_wls",
+    "gb_nir",
+}
```

Comment:
Also double-checked once more with Michael that it is okay to remove this, there's not specific privacy laws for these regions/countries within the UK 👍